bin/docker/shutdown_dev

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker rm -f discourse_dev
+docker stop discourse_dev && docker rm discourse_dev
```

Comment:
If I'm not mistaken, trying to gracefully stop the container will help propagate any delegated changes back to the host.